src/p5.dance.js

```diff
@@ -31,6 +32,7 @@ const WATCHED_RANGES = [0, 1, 2];
 
 const SIZE = constants.SIZE;
 const FRAMES = constants.FRAMES;
+const GENERATED_DANCER_SCALE = 0.75;
```

Comment:
For my understanding, why is this needed?

Reply:
Without it, the Lottie animation fills the entire canvas and the dancer looks quite a bit bigger than other sprites. At 75%, the size looks identical. I'm assuming that we scaled the original sprits down during the initial development in 2018 too, but I don't for sure. So admittedly, it's a bit of a magic number, but I'll add a comment.